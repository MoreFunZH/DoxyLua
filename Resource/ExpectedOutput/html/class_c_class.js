var class_c_class =
[
    [ "func1", "class_c_class.html#a8855ffb6d7e47584cfc78a04187991c1", null ],
    [ "func2", "class_c_class.html#aed01c6d6e0f00738dfa3ee9a4b6fdd86", null ],
    [ "prop1", "class_c_class.html#afc00978e2d2fba6703bab14704decf93", null ],
    [ "prop2", "class_c_class.html#a636bf08fa24b993a62438e4efa17daf4", null ]
];