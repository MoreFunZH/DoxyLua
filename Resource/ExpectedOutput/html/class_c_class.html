<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml">
<head>
<meta http-equiv="Content-Type" content="text/xhtml;charset=UTF-8"/>
<meta http-equiv="X-UA-Compatible" content="IE=9"/>
<meta name="generator" content="Doxygen 1.8.6"/>
<title>Sample Project: CClass Class Reference</title>
<link href="tabs.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="jquery.js"></script>
<script type="text/javascript" src="dynsections.js"></script>
<link href="navtree.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="resize.js"></script>
<script type="text/javascript" src="navtree.js"></script>
<script type="text/javascript">
  $(document).ready(initResizable);
  $(window).load(resizeHeight);
</script>
<link href="search/search.css" rel="stylesheet" type="text/css"/>
<script type="text/javascript" src="search/search.js"></script>
<script type="text/javascript">
  $(document).ready(function() { searchBox.OnSelectItem(0); });
</script>
<link href="doxygen.css" rel="stylesheet" type="text/css" />
</head>
<body>
<div id="top"><!-- do not remove this div, it is closed by doxygen! -->
<div id="titlearea">
<table cellspacing="0" cellpadding="0">
 <tbody>
 <tr style="height: 56px;">
  <td style="padding-left: 0.5em;">
   <div id="projectname">Sample Project
   </div>
   <div id="projectbrief">This is the sample output of DoxyLua.</div>
  </td>
   <td>        <div id="MSearchBox" class="MSearchBoxInactive">
        <span class="left">
          <img id="MSearchSelect" src="search/mag_sel.png"
               onmouseover="return searchBox.OnSearchSelectShow()"
               onmouseout="return searchBox.OnSearchSelectHide()"
               alt=""/>
          <input type="text" id="MSearchField" value="Search" accesskey="S"
               onfocus="searchBox.OnSearchFieldFocus(true)" 
               onblur="searchBox.OnSearchFieldFocus(false)" 
               onkeyup="searchBox.OnSearchFieldChange(event)"/>
          </span><span class="right">
            <a id="MSearchClose" href="javascript:searchBox.CloseResultsWindow()"><img id="MSearchCloseImg" border="0" src="search/close.png" alt=""/></a>
          </span>
        </div>
</td>
 </tr>
 </tbody>
</table>
</div>
<!-- end header part -->
<!-- Generated by Doxygen 1.8.6 -->
<script type="text/javascript">
var searchBox = new SearchBox("searchBox", "search",false,'Search');
</script>
</div><!-- top -->
<div id="side-nav" class="ui-resizable side-nav-resizable">
  <div id="nav-tree">
    <div id="nav-tree-contents">
      <div id="nav-sync" class="sync"></div>
    </div>
  </div>
  <div id="splitbar" style="-moz-user-select:none;" 
       class="ui-resizable-handle">
  </div>
</div>
<script type="text/javascript">
$(document).ready(function(){initNavTree('class_c_class.html','');});
</script>
<div id="doc-content">
<!-- window showing the filter options -->
<div id="MSearchSelectWindow"
     onmouseover="return searchBox.OnSearchSelectShow()"
     onmouseout="return searchBox.OnSearchSelectHide()"
     onkeydown="return searchBox.OnSearchSelectKey(event)">
<a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(0)"><span class="SelectionMark">&#160;</span>All</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(1)"><span class="SelectionMark">&#160;</span>Classes</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(2)"><span class="SelectionMark">&#160;</span>Files</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(3)"><span class="SelectionMark">&#160;</span>Functions</a><a class="SelectItem" href="javascript:void(0)" onclick="searchBox.OnSelectItem(4)"><span class="SelectionMark">&#160;</span>Variables</a></div>

<!-- iframe showing the search results (closed by default) -->
<div id="MSearchResultsWindow">
<iframe src="javascript:void(0)" frameborder="0" 
        name="MSearchResults" id="MSearchResults">
</iframe>
</div>

<div class="header">
  <div class="summary">
<a href="#pri-methods">Private Member Functions</a> &#124;
<a href="#pri-attribs">Private Attributes</a> &#124;
<a href="class_c_class-members.html">List of all members</a>  </div>
  <div class="headertitle">
<div class="title">CClass Class Reference</div>  </div>
</div><!--header-->
<div class="contents">

<p><code>#include &lt;<a class="el" href="sample1_8h_source.html">sample1.h</a>&gt;</code></p>
<table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pri-methods"></a>
Private Member Functions</h2></td></tr>
<tr class="memitem:a8855ffb6d7e47584cfc78a04187991c1"><td class="memItemLeft" align="right" valign="top">function&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="class_c_class.html#a8855ffb6d7e47584cfc78a04187991c1">func1</a> (arg1, arg2)</td></tr>
<tr class="separator:a8855ffb6d7e47584cfc78a04187991c1"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:aed01c6d6e0f00738dfa3ee9a4b6fdd86"><td class="memItemLeft" align="right" valign="top">function&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="class_c_class.html#aed01c6d6e0f00738dfa3ee9a4b6fdd86">func2</a> (arg1, arg2)</td></tr>
<tr class="separator:aed01c6d6e0f00738dfa3ee9a4b6fdd86"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table><table class="memberdecls">
<tr class="heading"><td colspan="2"><h2 class="groupheader"><a name="pri-attribs"></a>
Private Attributes</h2></td></tr>
<tr class="memitem:afc00978e2d2fba6703bab14704decf93"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="class_c_class.html#afc00978e2d2fba6703bab14704decf93">prop1</a> = nil</td></tr>
<tr class="memdesc:afc00978e2d2fba6703bab14704decf93"><td class="mdescLeft">&#160;</td><td class="mdescRight"><a class="el" href="class_this.html">This</a> is <a class="el" href="class_c_class.html">CClass</a>'s member property prop1.  <a href="#afc00978e2d2fba6703bab14704decf93">More...</a><br/></td></tr>
<tr class="separator:afc00978e2d2fba6703bab14704decf93"><td class="memSeparator" colspan="2">&#160;</td></tr>
<tr class="memitem:a636bf08fa24b993a62438e4efa17daf4"><td class="memItemLeft" align="right" valign="top">&#160;</td><td class="memItemRight" valign="bottom"><a class="el" href="class_c_class.html#a636bf08fa24b993a62438e4efa17daf4">prop2</a> = nil</td></tr>
<tr class="memdesc:a636bf08fa24b993a62438e4efa17daf4"><td class="mdescLeft">&#160;</td><td class="mdescRight"><a class="el" href="class_this.html">This</a> is <a class="el" href="class_c_class.html">CClass</a>'s member property prop2.  <a href="#a636bf08fa24b993a62438e4efa17daf4">More...</a><br/></td></tr>
<tr class="separator:a636bf08fa24b993a62438e4efa17daf4"><td class="memSeparator" colspan="2">&#160;</td></tr>
</table>
<a name="details" id="details"></a><h2 class="groupheader">Detailed Description</h2>
<div class="textblock">
<p>Definition at line <a class="el" href="sample1_8h_source.html#l00015">15</a> of file <a class="el" href="sample1_8h_source.html">sample1.h</a>.</p>
</div><h2 class="groupheader">Member Function Documentation</h2>
<a class="anchor" id="a8855ffb6d7e47584cfc78a04187991c1"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">function CClass::func1 </td>
          <td>(</td>
          <td class="paramtype">arg1&#160;</td>
          <td class="paramname">, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">arg2&#160;</td>
          <td class="paramname">&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">private</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<p><a class="el" href="class_this.html">This</a> is <a class="el" href="class_c_class.html">CClass</a>'s member function func1. </p>
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">arg1</td><td><a class="el" href="class_this.html">This</a> is arg1. </td></tr>
    <tr><td class="paramname">arg2</td><td><a class="el" href="class_this.html">This</a> is arg2. </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Return something. </dd></dl>

</div>
</div>
<a class="anchor" id="aed01c6d6e0f00738dfa3ee9a4b6fdd86"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">function CClass::func2 </td>
          <td>(</td>
          <td class="paramtype">arg1&#160;</td>
          <td class="paramname">, </td>
        </tr>
        <tr>
          <td class="paramkey"></td>
          <td></td>
          <td class="paramtype">arg2&#160;</td>
          <td class="paramname">&#160;</td>
        </tr>
        <tr>
          <td></td>
          <td>)</td>
          <td></td><td></td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">private</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">
<p><a class="el" href="class_this.html">This</a> is <a class="el" href="class_c_class.html">CClass</a>'s member function func2. </p>
<dl class="params"><dt>Parameters</dt><dd>
  <table class="params">
    <tr><td class="paramname">arg1</td><td><a class="el" href="class_this.html">This</a> is arg1. </td></tr>
    <tr><td class="paramname">arg2</td><td><a class="el" href="class_this.html">This</a> is arg2. </td></tr>
  </table>
  </dd>
</dl>
<dl class="section return"><dt>Returns</dt><dd>Return something. </dd></dl>

</div>
</div>
<h2 class="groupheader">Member Data Documentation</h2>
<a class="anchor" id="afc00978e2d2fba6703bab14704decf93"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">CClass::prop1 = nil</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">private</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

<p><a class="el" href="class_this.html">This</a> is <a class="el" href="class_c_class.html">CClass</a>'s member property prop1. </p>

<p>Definition at line <a class="el" href="sample1_8h_source.html#l00019">19</a> of file <a class="el" href="sample1_8h_source.html">sample1.h</a>.</p>

</div>
</div>
<a class="anchor" id="a636bf08fa24b993a62438e4efa17daf4"></a>
<div class="memitem">
<div class="memproto">
<table class="mlabels">
  <tr>
  <td class="mlabels-left">
      <table class="memname">
        <tr>
          <td class="memname">CClass::prop2 = nil</td>
        </tr>
      </table>
  </td>
  <td class="mlabels-right">
<span class="mlabels"><span class="mlabel">private</span></span>  </td>
  </tr>
</table>
</div><div class="memdoc">

<p><a class="el" href="class_this.html">This</a> is <a class="el" href="class_c_class.html">CClass</a>'s member property prop2. </p>

<p>Definition at line <a class="el" href="sample1_8h_source.html#l00021">21</a> of file <a class="el" href="sample1_8h_source.html">sample1.h</a>.</p>

</div>
</div>
<hr/>The documentation for this class was generated from the following file:<ul>
<li><a class="el" href="sample1_8h_source.html">sample1.h</a></li>
</ul>
</div><!-- contents -->
</div><!-- doc-content -->
<!-- start footer part -->
<div id="nav-path" class="navpath"><!-- id is needed for treeview function! -->
  <ul>
    <li class="navelem"><a class="el" href="class_c_class.html">CClass</a></li>
    <li class="footer">Generated on Mon Mar 31 2014 10:19:55 for Sample Project by
    <a href="http://www.doxygen.org/index.html">
    <img class="footer" src="doxygen.png" alt="doxygen"/></a> 1.8.6 </li>
  </ul>
</div>
</body>
</html>
